packages/emails/templates/_base-email.ts

```diff
@@ -74,7 +84,6 @@ export default class BaseEmail {
     return {
       transport: serverConfig.transport,
       from: serverConfig.from,
-      headers: serverConfig.headers,
```

Comment:
If it's not intentional. Let's undo this line delete.

Reply:
It's our mistake. We reverted it!